variables.tf

```diff
@@ -79,3 +79,62 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "cluster_ingress_hostname" {
+  type        = string
+  description = "Ingress hostname of the IKS cluster."
+  default     = ""
+}
+
+variable "operator_namespace" {
+  type        = string
+  description = "operator namespace"
+  default     = "cpd-operators"
+}
+
+variable "cpd_namespace" {
+  type        = string
+  description = "cpd namespace"
+  default     = "gitops-cp4d-instance"
+}
+
+variable "service_account_name" {
```

Comment:
Do we need to expose this variable? Is there a reason why someone would want to change this?

Reply:
implemented the change